agent/src/main/java/com/microsoft/applicationinsights/agent/internal/model/IncomingSpanImpl.java

```diff
@@ -209,8 +211,8 @@ private void finishBuildingTelemetry(long endTimeMillis) {
         try {
             requestTelemetry.setUrl(removeSessionIdFromUri(getUrl(detail)));
         } catch (MalformedURLException e) {
-            InternalLogger.INSTANCE.error("%s", e.toString());
-            InternalLogger.INSTANCE.trace("Stack trace is%n%s", ExceptionUtils.getStackTrace(e));
+            logger.error(e.getMessage());
```

Comment:
I want to stop using the `"Stack trace is"` log message pattern. I don't think it's useful. Here, I'd rather see `logger.error(e.getMessage(), e)`